cli.py

```diff
@@ -1936,29 +1936,29 @@ def returnGroupInfo(self, userlist):
             1 = we've errored, exit with error string
             2 = we've got work yet to do, onto the next stage
         """
-        for strng in userlist:
+        for string in userlist:
```

Comment:
Ah. I had tested that `string` wasn't in global scope -- I'm used to the `str` type.

Would `user` be ok?